examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -90,7 +90,9 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
       }
     }
 
-    function getFilter(parameters, maxDay, geoIds) {
+    const GetAllInLevel = 1;
```

Comment:
Should be a boolean value? And why do we need this?

Reply:
Yes, it should be boolean value.
This parameter allows me to request all units of the same selected level (e.x.county), without listing all countyIds.
I have a question about this choice of number in the original code: geoIds.length <= 2000?

Reply:
Is there a specific reason behind "2000" (an approximation of time spending)?

Reply:
I think it's about the size of "request". If it's bigger, the byte size becomes bigger, and the system was not able to send/receive it correctly because of the buffer size.

Reply:
Understood, thanks.